ui/appui-layout-react/src/appui-layout-react/widget/TabBar.tsx

```diff
@@ -67,9 +70,15 @@ export function WidgetTabBar(props: WidgetTabBarProps) {
           type: "WIDGET_DRAG_END",
           floatingWidgetId,
           target,
+          isActiveTabPreview: changeActiveTabAfterDragDrop,
```

Comment:
@GerardasB Given that the impact is not in a React component, would it make sense to change to a zustand hook in appui-layout-react as well so we dont have to add properties like that all over the place ?

Reply:
Let's keep the reducer pure. To avoid these properties we could create a reducer factory.